app/views/bulkrax/importers/index.html.erb

```diff
@@ -38,7 +38,7 @@
             <% @importers.each do |importer| %>
               <tr>
                 <th scope="row"><%= link_to importer.name, importer_path(importer) %></th>
-                <td><%= importer.status %></td>
+                <td><%= importer.last_run&.enqueued_records&.zero? ? 'Complete' : importer.status %></td>
```

Comment:
is this still needed now that `record_status` is being called on collection entries?

Reply:
I believe so, sometimes the works finish first and sometimes the collections finish first. I think that's why it felt so random when we were testing it.

Reply:
Ohh that part no